content/notebooks/mrchem-laptop-1.ipynb

```diff
@@ -0,0 +1,299 @@
+{
```

Comment:
<p>I think you can use LaTeX: *MW precision of $10^{-n}$*</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MRChemSoft/multiwavelet-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>